python/pyspark/pandas/data_type_ops/base.py

```diff
@@ -24,7 +24,7 @@
 import pandas as pd
 from pandas.api.types import CategoricalDtype
 
-from pyspark.sql import functions as F, Column
+from pyspark.sql import functions as F, Column as PySparkColumn
```

Comment:
Does this work? 
from pyspark.sql import functions as F
from pyspark.sql.column import Column as PySparkColumn